src/backend/distributed/operations/worker_node_manager.c

```diff
@@ -499,6 +499,18 @@ DistributedTablePlacementNodeList(LOCKMODE lockMode)
 }
 
 
+bool
+IsSingleNodeCluster(void)
```

Comment:
Even though I don't think it is necessary here, the convention is to have a comment